examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OverviewViewModel.kt

```diff
@@ -80,6 +84,19 @@ class OverviewViewModel(private val interactionHandler: OverviewInteractionHandl
         }
     }
 
+    fun retrieveCustomerInfo() {
+        viewModelScope.launch {
+            runCatching {
+                interactionHandler.customerInfo(Purchases.sharedInstance.awaitCustomerInfo())
```

Comment:
Hmm should we set the view model customer info live data to this value here instead? Seems like this could happen internally, no need to use the `interactionHandler`.

Reply:
fixed in [f129d2e](https://github.com/RevenueCat/purchases-android/pull/1012/commits/f129d2ed6039817325a4178756c397b79e17ca36)